tests/integration/security/rbac/rbac_v2_test.go

```diff
@@ -38,23 +40,27 @@ const (
 )
 
 var (
-	inst                 istio.Instance
-	successIfAuthEnabled bool
+	inst               istio.Instance
+	allowIfAuthEnabled bool
 )
 
 func TestMain(m *testing.M) {
-	framework.Main("rbac_v2_test", m, istio.SetupOnKube(&inst, setupConfig))
+	framework.
+		NewSuite("rbac_v2", m).
+		Label(label.Presubmit).
```

Comment:
Shouldn't be presubmit until it's stable (no failures for a week)

Reply:
@yangminzhu thoughts?

Also, fyi https://github.com/istio/istio/issues/13390

Reply:
Will the test run without `presubmit` label?

In the previous PR, the test is skipped: https://github.com/istio/istio/pull/13353#pullrequestreview-226934359

Reply:
It runs in prow not circleci. @ozevren

Reply:
Okay, then we can wait until it's stable.

Reply:
Okay then I will remove the presubmit label for now.